src/ensembl/src/shared/components/species-tabs-wrapper/MultiLineSpeciesWrapper.scss

```diff
@@ -1,5 +1,6 @@
 .linkWrapper {
   margin-left: 7px;
-  vertical-align: middle;
+  margin-top: 6px;
   white-space: nowrap;
+  float: left;
```

Comment:
Same as above; surely we can do better than floats?

Reply:
Removed it